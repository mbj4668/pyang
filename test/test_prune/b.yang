module b {
  yang-version 1.1;
  namespace urn:b;
  prefix b;

  import a {
    prefix a;
  }

  feature foo;
  feature bar;

  container a {
    if-feature "foo and not bar or a:foo";
  }

  container b {
    if-feature "foo and not bar";
  }

  container c {
    if-feature "foo and bar";
  }

  container d {
    if-feature "not bar and a:fox";
  }

  container e {
    if-feature "not bar and a:foz";
  }

}
