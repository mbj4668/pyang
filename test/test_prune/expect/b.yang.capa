urn:b?module=b&features=foo
