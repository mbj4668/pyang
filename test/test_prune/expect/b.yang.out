module: b
  +--rw a {foo and not bar or a:foo}?
  +--rw b {foo and not bar}?
  +--rw d {not bar and a:fox}?
